mw-webapp/src/service/CurrentProblemService.ts

```diff
@@ -1,16 +1,19 @@
-import {get, ref} from "firebase/database";
-import {CurrentProblemDTOToCurrentProblemConverter} from "src/converter/CurrentProblemConverter";
+import {collection, getDocs} from "firebase/firestore";
+import {
+  currentProblemDTOToCurrentProblemConverter,
+  querySnapshotToCurrentProblemDTOConverter,
+} from "src/converter/currentProblemConverter";
 import {db} from "src/firebase";
 import {CurrentProblem} from "src/model/businessModel/CurrentProblem";
 import {CurrentProblem as CurrentProblemDTO} from "src/model/firebaseCollection/CurrentProblem";
+import {PathToCollection} from "src/service/PathToCollection";
 
 export class CurrentProblemService {
 
-  public static async onValueFromRealTimeDb(): Promise<CurrentProblem[]> {
-    const snapshot = await get(ref(db, "/currentProblems"));
-    const currentProblemsRaw: CurrentProblemDTO[] = await snapshot.val();
-    const currentProblems: CurrentProblem[] = currentProblemsRaw.map((item) =>
-      CurrentProblemDTOToCurrentProblemConverter(item));
+  public static async getCurrentProblems(): Promise<CurrentProblem[]> {
+    const currentProblemsRaw = await getDocs(collection(db, PathToCollection.currentProblems));
+    const currentProblemsDTO: CurrentProblemDTO[] = querySnapshotToCurrentProblemDTOConverter(currentProblemsRaw);
+    const currentProblems = currentProblemsDTO.map(currentProblemDTOToCurrentProblemConverter);
```

Comment:
why do we have *Raw, *DTO and *? Why not to use 2 of them?

Reply:
Done